doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -58,8 +58,8 @@ Installing Payara
 
 - Download and install Payara (installed in ``/usr/local/payara5`` in the example commands below)::
 
-	# wget https://github.com/payara/Payara/releases/download/payara-server-5.2020.2/payara-5.2020.2.zip
-	# unzip payara-5.2020.2.zip
+	# wget https://s3-eu-west-1.amazonaws.com/payara.fish/Payara+Downloads/5.2020.6/payara-5.2020.6.zip
+	# unzip payara-5.2020.6.zip
```

Comment:
Can you please update this in the dev guide too? Here:

doc/sphinx-guides/source/developers/dev-environment.rst
88:``sudo curl -O -L https://github.com/payara/Payara/releases/download/payara-server-5.2020.2/payara-5.2020.2.zip``
90:``sudo unzip payara-5.2020.2.zip``

Reply:
Can you please update this in Vagrant too? Here:

scripts/vagrant/setup.sh
46:PAYARA_ZIP="$DOWNLOAD_DIR/payara-5.2020.2.zip"

And here:

downloads/download.sh
2:curl -L -O https://github.com/payara/Payara/releases/download/payara-server-5.2020.2/payara-5.2020.2.zip

Reply:
Can you please update this for docker-aio too? Here:

conf/docker-aio/c8.dockerfile
21:RUN cd /opt ; unzip /tmp/dv/deps/payara-5.2020.2.zip ; ln -s /opt/payara5 /opt/glassfish4

conf/docker-aio/testdata/scripts/vagrant/setup.sh
46:PAYARA_ZIP="$DOWNLOAD_DIR/payara-5.2020.2.zip"

conf/docker-aio/0prep_deps.sh
7:if [ ! -e dv/deps/payara-5.2020.2.zip ]; then
10:	wget https://github.com/payara/Payara/releases/download/payara-server-5.2020.2/payara-5.2020.2.zip  -O dv/deps/payara-5.2020.2.zip

Reply:
All done.

Reply:
@poikilotherm sure. It looks like you missed this one:

conf/docker-aio/testdata/scripts/vagrant/setup.sh
46:PAYARA_ZIP="$DOWNLOAD_DIR/payara-5.2020.2.zip"

I'm not actually sure if it's used or not. @pameyer might know.

Reply:
@pdurbin I'm sorry, but this file is not part of the git repo. Maybe you are symlinking somewhere during docker-aio setup?

Reply:
@poikilotherm ah, you're right, `git ls-files conf/docker-aio/testdata/scripts/vagrant/setup.sh` returns empty so there's nothing to fix or change. Thanks.

Reply:
@pdurbin already figured it out - `conf/docker-aio/testdata` is used as a temporary cache during image build.